raiden/ui/cli.py

```diff
@@ -405,11 +405,17 @@ def options(func):
             ),
             option(
                 "--proportional-fee",
-                help=("Mediation for as ratio of mediated amount in parts-per-million (10^-6)"),
+                help=("Mediation fee as ratio of mediated amount in parts-per-million (10^-6)."),
                 default=0,
                 type=int,
                 show_default=True,
             ),
+            option(
+                "--rebalancing-fee/--no-rebalancing-fee",
```

Comment:
If you want to allow a bit more configurability, you could pass in the parabola exponent instead of just an on/off flag.

Reply:
Let's put that in a later PR.